### PR TITLE
update kin core - lazy init libsodium, multitouch on marketplace false

### PR DESCRIPTION
**Pull request by OrenZak**
#### Main purpose:
- Update kin core to lazy init libsodium
- catch libsodium load lib exception and show error
- disable multitouch on ecosystem main activity

### CODE REVIEW

File: sdk/src/main/res/layout/kinecosystem_activity_main.xml [hunk 1]
```diff
@@ -5,6 +5,7 @@
     android:layout_width="match_parent"
     android:layout_height="match_parent"
     android:orientation="vertical"
+    android:splitMotionEvents="false"
```
Comment (YohayBar) [inline sdk/src/main/res/layout/kinecosystem_activity_main.xml:8]: this is the default setting?

File: dependencies.gradle [hunk 2]
```diff
@@ -1,7 +1,7 @@
 ext {
 
     //Version
-    kinCoreVersion = '3907369a9c'
+    kinCoreVersion = '95645e1c17'
```
Comment (YohayBar) [inline dependencies.gradle:4]: 95645e1c17 parent is not 3907369a9c?

Comment (OrenZak) [reply dependencies.gradle:4]: no

File: sdk/src/main/res/layout/kinecosystem_activity_main.xml (hunk 1 above)
Comment (OrenZak) [reply sdk/src/main/res/layout/kinecosystem_activity_main.xml:8]: what do you mean by default ?

**Review by YohayBar — APPROVED** [review]

**Pull request merged**